nodejs/week1/src/backend/main.js

```diff
@@ -0,0 +1,84 @@
+// import data here
+const meals = require("./data/meals");
+const reservations = require("./data/reservations.json");
+const reviewsjson = require("./data/reviews.json");
+
+module.exports = {
+  // return all the meals (including it's reviews)
+  getAllMealsIncludedReiews: () => {
+    //create a deep copy of Meals before including reviews in it.
+    const allMeals = JSON.parse(JSON.stringify(meals));
+    const allMealsIncludedReiews = allMeals.map((meal) => {
+      meal.reviews = [];
+      reviewsjson.forEach((review) => {
+        if (review.mealId === meal.id) {
+          meal["reviews"].push(review);
+        }
+      });
+      return meal;
+    });
+    return allMealsIncludedReiews;
+  },
+  /**return all the meals (including it's reviews)
+   *  that are cheap (you define what a cheap meal is) */
+  getAllCheapMealsIncludedReiews: () => {
+    //create a deep copy of Meals before including reviews in it.
+    const allMeals = JSON.parse(JSON.stringify(meals));
+    const allCheapMealsIncludedReiews = allMeals
+      .filter((currentMeal) => currentMeal.price < 69)
+      .map((meal) => {
```

Comment:
Maybe it's better to create a global variable for meals, and you can avoid repeating it on functions.

Reply:
ok, thanks